master/getting-started/docker/tutorials/advanced-label-policy.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Advanced Container Labels Policy
+---
+
+## Background
+
+With Calico as a Docker network plugin, Calico can be configured to extract the
+labels on a container and apply them to the workload endpoint for use with
+Calico
+[policy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/policy).
+When Calico is configured to use Container Labels, profiles are not created and 
```

Comment:
I would probably add the detail about the profile as a separate para after this one - so we can spell out that the default is full isolation between containers until policy is added.